spec/MyArraySpec.js

```diff
@@ -203,7 +207,7 @@ describe('MyArray', function () {
         it('returns index of the only occurrence', function () {
             var array = MyArray.of(1, '2', 2, 4);
 
-            expect(array.indexOf(2)).toEqual(2);
+            expect(array.indexOf(4)).toEqual(3);
```

Comment:
Why this change? It was better previously, because it ensures that the right comparator is used. E.g. you can spot bad code here if `array.indexOf(2)` equals 1.

Reply:
To be precise, there was a problem when you can pass the test just by `if (element === 2) { .. profit .. }`